public/app/percona/dbaas/components/DBCluster/EditDBClusterModal/EditDBClusterModal.tsx

```diff
@@ -88,16 +90,16 @@ export const EditDBClusterModal: FC<EditDBClusterModalProps> = ({
   return (
     <div className={styles.modalWrapper}>
       <Modal title={editModalTitle} isVisible={isVisible} onClose={() => setVisible(false)}>
+        { showUnsafeConfigurationWarning && <UnsafeConfigurationWarning /> }
```

Comment:
Removed spaces around expression here.

Reply:
This may be while the unit tests failed as there are some lint errors.

Reply:
I tried it different ways today and it strange becouse my linter doesn't see this errors =(